storybookjs/storybook
# CI: Use a new cadence (ci, pr, merged, daily) and trigger CircleCI from github actions

## What I did

- Made a new cadence: ci, pr, merged, daily
- Trigger those cadences from github actions
- Labels can be used to override the default behavior

## How to test

- Try adding labels to this PR, and see it being triggered
- Move the PR to draft, and back to ready for review
- Trigger workflows from the circle ci API or web UI by passing workflow is `ci`, `pr`, `merged` or `daily`
